merge_coda_files/merge_coda_files.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import time
+from os import path
+import json
+from dateutil.parser import isoparse
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCoda2IO
+from core_data_modules.util import IOUtils
+from core_data_modules.data_models import Scheme
+
+# opens a coding scheme
+def open_scheme(filepath):
+    with open(filepath, "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+def get_strings_coda(key_map, code_scheme, list_td):
+    list_codeids = [code.code_id for code in code_scheme.codes]
```

Comment:
I think `code_ids_in_scheme` would be a better name

Reply:
In which case:
 - code ids is two words
 - include the "of"
 - I still think appending `in_scheme` is helpful. Otherwise it's not clear what's in the list - all you're giving is a data type

Reply:
Actually, given that code_ids is plural, I still think list_of_ is redundant

Reply:
changed to list_code_ids